code/game/objects/items/melee/baton.dm

```diff
@@ -210,7 +218,10 @@
 			var/mob/living/carbon/human/human_target = target
 			if(prob(force_say_chance))
 				human_target.force_say()
-		target.apply_damage(stamina_damage, STAMINA)
+		var/effective_armour_penetration = get_stun_penetration_value()
+		var/chest = target.get_bodypart(BODY_ZONE_CHEST)
```

Comment:
Not an issue, but do we really only want to check the chest? Could check the entire body instead